concrete/src/Csv/Csv.php

```diff
@@ -1,53 +0,0 @@
-<?php
```

Comment:
Do we need to shim any of the removed classes? should we just leave them and mark them as deprecated?